app/views/posts/index.html.erb

```diff
@@ -1 +1,40 @@
 <h2>Here is a list of all the post for a particular user</h2>
+<section id="each_user_allpost">
+    <%= render "shared/user", user: @user %>
+    <br>
+
+    <div class="all_post">
+        <% @user_posts.each do |post|%>
+            <div class="each_post">
+                <div class="post_details">
+                    <h3><%= post.title%></h3>
+                    <p class="post_text"><%= post.text %></p>
+                    <p class="comments_likes_count">
+                        Comments: <%= post.comments_counter || 0%>,
+                        Likes: <%= post.likes_counter || 0%>
+                    </p>
+                    <%= link_to "view post", user_post_path(@user, post) %>
+                </div>
+
+                <div class="post_comments">
+                    <h4>Comments</h4>
+                    <% @post_comments.each do |comment| %>
+                        <ul>
+                            <% if comment.post_id === post.id %>
+                                <li><%= comment.author.name%>: <%= comment.text %></li>
+                            <% end %>
+                        </ul>
+
+                    <% end %>
+
+                </div>
```

Comment:
- [ ] It is expected that you use the `post.five_recent_comments`which you already defined in the `Post` model to render 5 most recent comments in the `app/views/posts/index.html.erb` page 👍 

### Expected
![image](https://github.com/oguntayomicheal/Blogs-App/assets/46839548/34f0cfcb-2b8b-4413-80ba-e5c4b2ddcdea)
### Actual
![image](https://github.com/oguntayomicheal/Blogs-App/assets/46839548/6dfe7e97-43d3-416c-85d6-ced6bc9ddf7f)